### DISCUSSION TITLE
Reducing Power Consumption for remote use

### DISCUSSION THREAD

**Discussion by Vollpflock** [mcguirepr89/BirdNET-Pi#1108] [Q&A] [answered]
I purchased a Raspberry Pi 5, anticipating it would consume less power, only to find that it uses around the same amount of energy as the Raspberry Pi 4 model. I'm looking for ideas on how to use it remotely with reduced energy consumption. Any ideas?

Pi 4 with external HDD: 6.5 watts

Pi 5 without external disk: 3.8 watts
Pi 5 with external HDD: 5.5 watts
Pi 5 with external SSD: 6.3 watts

**Comment by jalbarra** [comment, marked as answer]
If you use a Raspberry pi 3 surely you will have less power consumption.
It's the better solution for autonomous stations
